src/Splitio/Domain/MatcherDefinition.cs

```diff
@@ -13,5 +13,6 @@ public class MatcherDefinition
         public DependencyData dependencyMatcherData { get; set; }
         public bool? booleanMatcherData { get; set; }
         public string stringMatcherData { get; set; }
+        public BetweenStringData BetweenStringMatcherData { get; set; }
```

Comment:
Just a code style comment: use lowercase (`betweenStringMatcherData`) like the other properties.